pydbgen/pydbgen.py

```diff
@@ -45,266 +69,151 @@ def license_plate(self,seed=None,style=None):
-        elif style==2:
-            result=''
-            for _ in range(3):
-                result+=chr(randint(65,90))
-            result+='-'
-            for _ in range(4):
-                result+=str(randint(0,9))
-            return result
+        if not style:
+            style = choice([1, 2, 3])
+
+        license_place_format = "{p1}{p2}{p3}"
+
+        if style == 1:
+            p1 = str(randint(1, 9))
+            p2 = "".join([chr(randint(65, 90)) for _ in range(3)])
+            p3 = "".join([str(randint(1, 9)) for _ in range(3)])
+        elif style == 2:
+            p1 = "".join([chr(randint(65, 90)) for _ in range(3)])
+            p2 = "-"
+            p3 = "".join([str(randint(0, 9)) for _ in range(4)])
         else:
-            result=''
-            for _ in range(3):
-                result+=chr(randint(65,90))
-            result+='-'
-            for _ in range(3):
-                result+=str(randint(0,9))
-            return result
+            p1 = "".join([chr(randint(65, 90)) for _ in range(3)])
+            p2 = "-"
+            p3 = "".join([str(randint(0, 9)) for _ in range(3)])
 
+        return license_place_format.format(p1=p1, p2=p2, p3=p3)
 
-    def realistic_email(self,name,seed=None):
+
+    def realistic_email(self, name, seed=None):
         '''
         Generates realistic email from first and last name and a random domain address
         seed: Currently not used. Uses seed from the pydb class if chosen by user
         '''
-        import random
-        import os
-        from random import randint,choice
         random.seed(self.seed)
 
-        name=str(name)
-        result=''
+        name = str(name)
         f_name = name.split()[0]
         l_name = name.split()[-1]
-        
+
         choice_int = choice(range(10))
-        
-         
-        dir_path = os.path.dirname(os.path.realpath(__file__))
-        print(path)
-        path = dir_path+"\Domains.txt"
-        
-        domain_list = []
-        fh = open(path)
-        for line in fh.readlines():
-            domain_list.append(str(line).strip())
-
-        domain = choice(domain_list)
-        fh.close()       
-        
-        name_choice = choice(range(8))
-        
-        if name_choice==0:
-            name_combo=f_name[0]+l_name
-        elif name_choice==1:
-            name_combo=f_name+l_name
-        elif name_choice==2:
-            name_combo=f_name+'.'+l_name[0]
-        elif name_choice==3:
-            name_combo=f_name+'_'+l_name[0]
-        elif name_choice==4:
-            name_combo=f_name+'.'+l_name
-        elif name_choice==5:
-            name_combo=f_name+'_'+l_name
-        elif name_choice==6:
-            name_combo=l_name+'_'+f_name
-        elif name_choice==7:
-            name_combo=l_name+'.'+f_name
-        
-        if (choice_int<7):
-            result+=name_combo+'@'+str(domain)
+
+        domain = choice(self.domain_list)
+
+        name_formats = ["{f}{last}", "{first}{last}",

```

Comment:
This allows us to easily add additional formats and avoids the long if-else ladder.